content/blog/purrr-1-2-0/index.Rmd

```diff
@@ -0,0 +1,93 @@
+---
+output: hugodown::hugo_document
+
+slug: purrr-1-2-0
+title: purrr 1.2.0
+date: 2025-10-27
+author: Hadley Wickham
+description: >
+    This release tightens up the package by removing long-deprecated functions 
+    and making `map_chr()` and predicate functions more type-safe. It also 
+    includes performance improvements to `every()`, `some()`, and `none()`, 
+    as well as a new getting started vignette.
+
+photo:
+  url: https://unsplash.com/photos/orange-tabby-kitten-in-grasses-RCfi7vgJjUY
+  author: Andriyko Podilnyk
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [purrr]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+We're thrilled to announce the release of purrr 1.2.0! purrr enhances R's functional programming toolkit with a complete and consistent set of tools for working with functions and vectors.
+
+You can install it from CRAN with:
+
+```{r}
+#| eval: false
+install.packages("purrr")
+```
+
+Overall, this is a pretty unexciting release since it primarily focusses on removing long-deprecated functions. It does, however, include a couple of small performance improvements to predicate functions and a brand new getting started vignette. You can see a full list of changes in the [release notes](https://github.com/tidyverse/purrr/releases/tag/v1.2.0).
```

Comment:
```suggestion
Overall, this is a pretty unexciting release since it primarily focuses on removing long-deprecated functions. It does, however, include a couple of small performance improvements to predicate functions and a brand new getting started vignette. You can see a full list of changes in the [release notes](https://github.com/tidyverse/purrr/releases/tag/v1.2.0).
```